packages/backend-output-schemas/API.md

```diff
@@ -275,24 +275,29 @@ export const unifiedBackendOutputSchema: z.ZodObject<{
         payload: z.ZodObject<{
             bucketName: z.ZodString;
             storageRegion: z.ZodString;
+            buckets: z.ZodOptional<z.ZodArray<z.ZodRecord<z.ZodString, z.ZodString>, "many">>;

```

Comment:
why typing for `buckets` is different here than in other places ?

additionally. should we use records here or should be come up with subtype with named properties? (this applies to all `buckets` introduced in this file)

Reply:
Done d67854ec809441ddd20bc88407be944aa6950d42